src/ChunkMap.cpp

```diff
@@ -740,17 +740,17 @@ bool cChunkMap::DoWithChunkAt(Vector3i a_BlockPos, std::function<bool(cChunk &)>
 
 
 
-void cChunkMap::WakeUpSimulators(int a_BlockX, int a_BlockY, int a_BlockZ)
+void cChunkMap::WakeUpSimulators(const Vector3i & a_Block)
 {
 	cCSLock Lock(m_CSChunks);
 	int ChunkX, ChunkZ;
-	cChunkDef::BlockToChunk(a_BlockX, a_BlockZ, ChunkX, ChunkZ);
+	cChunkDef::BlockToChunk(a_Block.x, a_Block.z, ChunkX, ChunkZ);
```

Comment:
Thinking about it more, I feel like using Vector3 in place of a Vector2 is much worse in terms of code readability (consider e.g. cChunkMap::MarkChunkDirty, or any cChunkMap - is there a "Y coord is ignored" comment for every function?). Unless there's an objection I'll add a Vector2 class and use that where appropriate.

Vector2 shouldn't be complicated, since Vector3 is fairly short and Vector2 will have slightly fewer methods.